docs/_templates/layout.html

```diff
@@ -27,4 +27,61 @@
       };
     })
   </script>
+  <style>
+    .example-header {
+      position: relative;
+      background: #9AAA7A;
+      padding: 8px 150px 8px 16px;
+      margin-bottom: 0;
+    }
+    .example-header:after{
+      content: '';
+      display: table;
+      clear: both;
+    }
+    .example-title {
+      display:block;
+      padding: 4px;
+      color: white;
+    }
+    .example-header-button {
+      top: 8px;
+      right: 16px;
+      position: absolute;
+    }
+    .example-header + .highlight-python {
+      margin-top: 0 !important;
+    }
+    .viewcode-button {
```

Comment:
I think it could be much nicer if the link to sources looks similar to how it looks in generated autoapi documentation "[source]" link rather than [View code] button. It should also be checked on a mobile screen. Current version looks pretty bad on mobile( [View code] button hides the actual name of file.

Actually I think there is a small problem that we cannot link to the actual line of code - only to the whole file, therefore the "View code" or "source" link can be misleading (because you think you go directly to the code that you see). In this case maybe instead of button or "source" link, the whole file name could be link on its own ? This suggests that you go to the file rather to the exact line of code.

Reply:
I have not tested it on a mobile device yet. I wanted to first collect opinions on how this should look. 
The appearance of this component was based on how Google did it.
![image](https://user-images.githubusercontent.com/12058428/53689823-b25a6700-3d5e-11e9-8d84-c215b5c679b1.png)
Personally, I think it looks better than just a simple link.

I can prepare a variant that does not include a button but a clickable filename. These visual issues should be discussed in the community.

I know about this problem and it's possible to fix it, but it's going to make me write twice more code. For now, I thought it was enough. Google also does not link to a lines in the file.

Reply:
Google have other variant of these component with link with filename and button :
![screenshot 2019-03-08 at 16 04 25](https://user-images.githubusercontent.com/12058428/54036486-08466900-41bc-11e9-81e9-cf09ca4362ec.png)